packages/rest/src/validation/request-body.validator.ts

```diff
@@ -51,7 +51,8 @@ export function validateRequestBody(
   if (!schema) return;
 
   const jsonSchema = convertToJsonSchema(schema);
-  validateValueAgainstJsonSchema(body, jsonSchema, globalSchemas);
+  const schemaTitle = schema.title || '';
```

Comment:
Can title ever be `undefined` in our case? Temporarily doing `schema.title || ''` to stop TS from complaining, this will be fixed later.

Reply:
See https://json-schema.org/understanding-json-schema/reference/generic.html#metadata. I think `title` is optional.

Reply:
We probably should generate uuid as the title to make it a unique key for caching.